packetbeat/tests/system/packetbeat.py

```diff
@@ -1,12 +1,12 @@
+from beat.beat import Proc
+from beat.beat import TestCase
 import os
 import sys
 import subprocess
 import json
 
 sys.path.append(os.path.join(os.path.dirname(__file__), '../../../libbeat/tests/system'))
 
-from beat.beat import TestCase
-from beat.beat import Proc
```

Comment:
Can you revert this change? It is causing the tests to fail because its looking for `beat.beat` on the wrong path.

Reply:
All done, and it looks like that build stage is passing now.